opentelemetry-sdk/tests/test_util.py

```diff
@@ -0,0 +1,210 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import unittest
+
+from opentelemetry.sdk.util import BoundedDict, BoundedList
+
+
+class TestBoundedList(unittest.TestCase):
+    base = [52, 36, 53, 29, 54, 99, 56, 48, 22, 35, 21, 65, 10, 95, 42, 60]
+
+    def test_raises(self):
+        with self.assertRaises(ValueError):
+            BoundedList(-1)
+
+        blist = BoundedList(4)
+        blist.append(37)
+        blist.append(13)
+
+        with self.assertRaises(IndexError):
+            _ = blist[2]
```

Comment:
is this assignment necessary? wouldn't __getitem__ get called regardless of what is done to the return value?

Reply:
I guess this way it's more explicit that we don't care about the value instead of having made a typo.

Reply:
pylint will complain with `W0104: Statement seems to have no effect (pointless-statement)` if `_ = ` is removed.